gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/rules/AnyOneOfExecutionEngineRule.java

```diff
@@ -59,6 +62,11 @@ protected ExecutionEngine findExecutionEngine(DMLQueryInfo qInfo, ExecutionRuleC
         return ExecutionEngine.SPARK;
       }
     }
+
+    List<TableQueryInfo> tqis = qInfo.getTableQueryInfoList();
```

Comment:
Add a comment here what this means. This class can use a general javadoc stating the rules being applied here for routing.